webpack.config.js

```diff
@@ -63,8 +71,8 @@ module.exports = draftConfig => {
     },
 
     output: {
-      path: path.resolve(__dirname, '/'),
-      publicPath: '/',
+      path: path.resolve('.', 'draft-build'),
+      publicPath: process.env.PUBLIC_PATH || '/',
```

Comment:
I feel like this is wrong. Or are we permanently adding a `./draft-build` folder in every project that consumes `draft`?

Reply:
The output path only applies if we write to disk, which only happens if env WRITE_TO_DISK is true. Otherwise it's stored in memory, which uses its own path stuff.